clinicadl/train/list_models_cli.py

```diff
@@ -0,0 +1,33 @@
+import click
+
+
+@click.command(name="list_models")
+@click.option(
+    "-a",
+    "--architecture",
+    type=str,
+    help="Architecture of the chosen model to display.",
+)
+@click.option(
+    "-in",
+    "--input_size",
+    type=int,
+    multiple=2,
+    default=(128, 128),
+    show_default=True,
+    help="Size of the input image.",
```

Comment:
In order to work with the 3D case, it could be better to use `nargs=True` and have a default value including the number of channels (which may be 1 or 3 depending on the slice setting).
```suggestion
    nargs=True,
    default=(1, 128, 128),
    show_default=True,
    help="Size of the input image.",
```